### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by gillessylvestre** [pnp/sp-dev-fx-webparts#2025] [General]
Hey all.

I'm not a developer and I would like a step-by-step guide on hoe to deploy the web part to SharePoint 365.

Any help would be appreciate.

Regards,
Gilles

**Comment by carry2web** [comment]
I think everything is rather well explained in step-by-step guide here: https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part
